src/spyglass/spikesorting/analysis/v1/group.py

```diff
@@ -0,0 +1,135 @@
+import datajoint as dj
+import numpy as np
+from ripple_detection import get_multiunit_population_firing_rate
+
+from spyglass.common import Session  # noqa: F401
+from spyglass.spikesorting.spikesorting_merge import SpikeSortingOutput
+from spyglass.spikesorting.unit_inclusion_merge import UnitInclusionOutput
+from spyglass.utils.dj_mixin import SpyglassMixin
+
+schema = dj.schema("sorted_spikes_group_v1")
```

Comment:
Is 'sorted' a new prefix for which everyone should be granted access? If so, please add to the list in `database_settings.py`

`waveform` snuck in on a prev-PR and I've been meaning to put together SQL roles to reflect our structure in `database-settings.py` for member/collab/outsider to make it easier to grant all members relevant permission on new shared modules.

Reply:
This was a mistake. I think waveform was also a mistake unfortunately. I changed this back.